src/walk.rs

```diff
@@ -66,6 +66,13 @@ pub fn scan(paths: &[PathBuf], patterns: Arc<Vec<Regex>>, config: Arc<Config>) -
             .add(pattern)
             .map_err(|e| anyhow!("Malformed exclude pattern: {}", e))?;
     }
+
+    if config.read_vcsignore {
+        override_builder
+            .add("!.git")
+            .map_err(|e| anyhow!("Malformed default exclude pattern: {}", e))?;
```

Comment:
Since we are directly providing the pattern here, and know that it is a valid pattern, I think it would be more appropriate to use expect here:

```suggestion
           .expect("Invalid exclude pattern"));
```

Reply:
Yup, that makes sense